DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/compareExchange/indexCompareExchange/IndexCompareExchange.java

```diff
@@ -33,7 +33,7 @@ public void setUnitsInStock(int unitsInStock) {
         //region Step_3
         public Products_ByUnitsInStock() {
             map = "docs.Products.Select(product => new {\n" +
-                "    UnitsInStock = this.LoadCompareExchangeValue(Id(product))\n" +
+                "    unitsInStock = this.LoadCompareExchangeValue(Id(product))\n" +
```

Comment:
are you sure about this change? In Java we are using custom conventions for casing.

Reply:
IndexCompareExchangeTest seems to be working w/o this change.